test/server.js

```diff
@@ -14,6 +14,7 @@ describe('Same-Realm Server Tests', function() {
     assert(isPlainObject(Object.create(Object.prototype)));
     assert(isPlainObject({foo: 'bar'}));
     assert(isPlainObject({}));
+    assert(isPlainObject(Object.create(null)));
```

Comment:
would this be a breaking change as it used to return false?

Also the readme still states it returns false
<img width="504" alt="image" src="https://user-images.githubusercontent.com/1569131/87998160-1108b800-caef-11ea-9d5e-c02f35a2c605.png">

Reply:
I bumped major version. Fixed readme. Thanks.